pkg/collector/corechecks/cluster/kubernetes_state_test.go

```diff
@@ -725,7 +690,7 @@ func TestKSMCheck_hostnameAndTags(t *testing.T) {
 					},
 				},
 			},
-			wantTags:     []string{"foo_label:foo_value", "foo_label:foo_value", "bar_label:bar_value", "baz_label:baz_value"},
+			wantTags:     []string{"foo_label:foo_value", "bar_label:bar_value", "baz_label:baz_value"},
```

Comment:
can we keep the key `foo_label` but we different values to make sure the label joiner adds both tags?

Reply:
No, it’s not possible because `foo_label` is a label to match. So, if it has a different value, it will not be used for label joins.
And, as labels are managed in a map, it’s not possible to have two values for a single key (one for matching and one to get)